.github/workflows/ci-app-tests.yml

```diff
@@ -105,6 +105,7 @@ jobs:
       - name: Adjust examples
         if: ${{ matrix.pkg-name != 'lightning' }}
         run: |
+          python .actions/assistant.py copy_replace_imports --source_dir="./tests" --source_import="lightning.app,lightning" --target_import="lightning_app,lightning_app"
```

Comment:
Is it expected that tests files use the new import style? Wouldn't it be better to update those usages?

Reply:
nope, that was my mistake. Has been updated